src/main/java/com/farao_community/farao/flow_decomposition/AbstractSparseMatrixWithIndexes.java

```diff
@@ -24,6 +24,6 @@ protected AbstractSparseMatrixWithIndexes(Map<String, Integer> rowIndex, Map<Str
     abstract Map<String, Map<String, Double>> toMap(boolean fillZeros);
 
     Map<String, Map<String, Double>> toMap() {
-        return toMap(false);
+        return toMap(FlowDecompositionResults.NOT_FILL_ZEROS);
```

Comment:
Create a copy of the constant directly in the class for readibility

Reply:
ok